### DISCUSSION TITLE
Specifying upstreams for domains:How to assign two upstreams to a domain?

### DISCUSSION THREAD

**Discussion by WoChen5770** [AdguardTeam/AdGuardHome#2789] [Q&A] [answered]
Thanks.How to assign two upstreams to a domain？
like：
[/github.com/]https://1.0.0.1/dns-query
[/github.com/]https://dns.google/dns-query
or:
[/github.com/]https://1.0.0.1/dns-query,https://dns.google/dns-query

**Comment by ameshkov** [comment, marked as answer]
This one:

```
[/github.com/]https://1.0.0.1/dns-query
[/github.com/]https://dns.google/dns-query
```

**Reply by WoChen5770** [reply]
thanks
